source/release-notes.txt

```diff
@@ -20,6 +20,31 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-9-0:
+
+|bi| 2.9.0
+----------
+
+*Released Dec 18, 2018*
+
+Features and Performance Improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Adds the ability to rewrite distinct aggregations as ``GROUP BY``
+  clauses to avoid document size limits.
+- Includes a sample ``mongosqld``
+  :ref:`configuration file <config-format>` in the release archive.
+- Allows setting commonly used system variables in the configuration
+  file.
+- Optimizes conversions of years to days.
+- Optimizes ``join`` performance.
+- General performance improvements.
```

Comment:
I added this because in the original ticket there was a bulleted list item in the JIRA ticket:

- Performance improvements